db/migrate/20210519122247_add_rating_to_rating.rb

```diff
@@ -0,0 +1,5 @@
+class AddRatingToRating < ActiveRecord::Migration[6.1]
```

Comment:
😄

Reply:
well we decided that rating can make use of some rating..